pyxem/signals/diffraction2d.py

```diff
@@ -704,8 +689,9 @@ def center_direct_beam(
         shifts=None,
         return_shifts=False,
         subpixel=True,
-        lazy_result=None,
+        lazy_output=None,
```

Comment:
While I agree that we should change `lazy_result` to `lazy_output` to conform to the parameter names in `s.map`. This might break already existing workflows.

I would rather add `lazy_output`, and keep `lazy_result` for the next release, but raise a deprecation warning if it is used. Warning that it will be removed in the next release.

Reply:
Sounds good! I'll add in depreciation